vue/wolox-books/src/views/Home.vue

```diff
@@ -0,0 +1,72 @@
+<template lang="pug">
+  .home-container
+    .navbar-container
+      .icon-container
+        img.wolox-icon-small(src='../assets/wolox_logo.svg' alt='Wolox logo')
+        p.icon-title
+          | BOOKS
+      a.logout(@click='logout()')
```

Comment:
this is a button

Reply:
As a summary, the difference between a button and a link is:
- A button is a tag that has an action that will be executed every time we click on it, so, it has an onClick prop that will trigger a function. This is commonly used to run some kind of js code to do something e.g. API calls, validations, triggering modifications to the DOM, etc.
- A link is only for routing purposes, this will have an attribute that will ref the route as a string.
Considering the above, we have here something that, the logout action will:
1. Remove the token from the localstorage
2. Redirect.
So,  if there's going to be some kind of logic inside this. The correct tag to the logout action, is a button.

Reply:
Thanks.